### ISSUE TITLE
Biosample links to Qiita

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#334]
The biosample class in the NMDC schema includes a variety of slots for capturing web-resolvable identifiers such as those from GOLD and INSDC.

Several of those identifiers are already materialized as links on the data portal

@emileyfadrosh asked if we could add links to biosamples on the Qiita site, especially for the EMP 500 samples.

@cmungall and I looked into that on Friday and did not see any per-sample pages at Qiita. So I am inclined to say no, we cant' do that.

**Comment by turbomam** [comment]
If we did find find Qiita per-sample pages, we would have to determine if they are parameterized with some IDs that we already have in mongodb. For example, if there's soemthing like a Qiita ID, we would either need to add that to the schema, or we would need to find a way to convert between the Qiita ID and soemthing we already do have for the biosamples.

**Comment by emileyfadrosh** [comment]
Correct, there are no per-sample pages on Qiita, just the study page: https://qiita.ucsd.edu/study/description/13114

@turbomam @cmungall would it make sense/be possible to add that as part of the study? thanks!

**Comment by turbomam** [comment]
Thanks, @emileyfadrosh 

When I enter https://qiita.ucsd.edu/study/description/13114 into Chrome, I am sent directly to the Qiita home page:

> Qiita Spots Patterns
Qiita (canonically pronounced cheetah) is an entirely open-source microbial study management platform...

**Comment by turbomam** [comment]
^ maybe because I'm not logged in!

**Comment by turbomam** [comment]
note to self: remember to log into Qiita!
